iOS/Flows/Room/RoomCoordinator+Notices.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  RoomCoordinator+Notices.swift
+//  Jibber
+//
+//  Created by Benji Dodgson on 4/1/22.
+//  Copyright © 2022 Benjamin Dodgson. All rights reserved.
+//
+
+import Foundation
+import StreamChat
+
+extension RoomCoordinator {
+    
+    // The primary action
+    func handleRightOption(with notice: SystemNotice) {
+        switch notice.type {
```

Comment:
Where we handle all interactions on a notice. Can only have 2 options. Most only have 1